codegen/java-gen/src/test/resources/tests/replicated-entity/multimap-absolute-packages/proto/multi_map_api.proto

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+import "akkaserverless/annotations.proto";
+
+package com.example.replicated.multimap;
+
+option java_outer_classname = "SomeMultiMapApi";
+
+message Key {
+  string key = 1;
+}
+
+message Value {
+  string value = 1;
+}
+
+message PutValue {
+  string multi_map_id = 1 [(akkaserverless.field).entity_key = true];
+  Key key = 2;
+  Value value = 3;
+}
+
+service MultiMapService {
+  option (akkaserverless.codegen) = {
+    replicated_entity: {
+      name: "com.example.replicated.multimap.domain.SomeMultiMap"
+      entity_type: "some-multi-map"
+      replicated_multi_map: {
+        key: "com.example.replicated.multimap.domain.SomeKey"
+        value: "com.example.replicated.multimap.domain.SomeValue"
```

Comment:
No version with primitive key or value?

Reply:
There is one, here: 
https://github.com/lightbend/akkaserverless-java-sdk/pull/780/files#diff-f95343914fb9bf1aaa9d2d6687ed719df48773ea5780b68f9a098b3e2809e8c2R38-R48